### PR TITLE
menu: add query parameters

### PR DESCRIPTION
**Pull request by Garfield-fr**
Allows to add parameters to the url when routerLink is active.

Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Extracted translations?

### CODE REVIEW

File: projects/rero/ng-core/src/lib/menu/menu-item.ts [hunk 1]
```diff
@@ -47,6 +47,9 @@ export class MenuItem implements MenuItemInterface {
   /** Uri */
   private _uri?: string;
 
+  /** Uri params */
+  private _QueryParams = {};
```
Comment (sebastiendeleze) [inline projects/rero/ng-core/src/lib/menu/menu-item.ts:51]: Should be `_queryParams` (minus `Q`)

**Review by sebastiendeleze — APPROVED** [review]

File: projects/rero/ng-core/src/lib/menu/menu-item.ts (hunk 1 above)
Comment (Garfield-fr) [reply projects/rero/ng-core/src/lib/menu/menu-item.ts]: done

**Review by iGormilhit — COMMENTED** [review]
You could use `parameters` instead of `params` in the commit title.

**Review by zannkukai — APPROVED** [review]

**Review by iGormilhit — APPROVED** [review]
Commit message approved.

**Pull request merged**